### ISSUE TITLE
[SAY][Suggestion] Server wide

### ISSUE THREAD

**Issue by Dragonias** [retke/Laggrons-Dumb-Cogs#162]
# Feature request

Cog: Say

## Description

Would it be possible to have a say command that lets me announce on other servers the Bot is in for example using the id and channel id.

**Comment by laggron42** [comment]
That is already possible if you use the say command in DM. However, it's not that stable (two channels can have the same ID).

Is what you want a way to make it work globally by provinding a server ID, making sure it gets sent in the right place?

**Comment by Dragonias** [comment]
Yes just by putting server id and channel id is basicly what I need as I sometimes announce things to certain servers each message being different so I cant just use a cog that just sends to "all" but I guess I can try that too  :)
